policyengine_us/variables/gov/states/va/tax/income/deductions/cdcc_expense/va_cdcc_exepense_deduction.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class va_cdcc_exepense_deduction(Variable):
```

Comment:
Please change variable names to avoid CDCC, which is a credit (last C)
```suggestion
class va_child_dependent_care_expense_deduction(Variable):
```

Reply:
ok just update changelog, issue title, pr title and we're good